app/src/main/java/com/ikarabulut/Game.java

```diff
@@ -0,0 +1,36 @@
+package com.ikarabulut;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Game {
+    private Board currentBoard;
+    private Board nextBoard;
+
+    public Game(Board board) {
+        currentBoard = board;
+        nextBoard = new Board(board.getRows(), board.getColumns());
+
+    }
+
+    public List<ArrayList> getNextBoard() {
+        return this.nextBoard.getBoard();
+    }
+
+    public void generateNextBoard() {
+        for (int row = 0; row < currentBoard.getRows(); row++) {
+            for (int column = 0; column < nextBoard.getColumns(); column++) {
+                int currentCellNumOfAliveNeighbors = currentBoard.getNumberOfAliveNeighbors(row, column);
+                boolean currentCellIsAlive = currentBoard.isCellAliveAt(row, column);
+                boolean nextGenerationStatus = Rules.determineFate(currentCellNumOfAliveNeighbors, currentCellIsAlive);
```

Comment:
I'd rename this variable to `shouldBeAliveNextGeneration` or something similar. It's not really the "status" of the cell, it's more the answer to the question "should this be alive?"

Reply:
This makes more sense. I applied this change since it was easy